sample_code/ep113/README.md

```diff
@@ -0,0 +1,45 @@
+# [don't put passwords in commands! (beginner - intermediate)](https://youtu.be/SqXOchNKV_c)
+
+Today I talk about why it's a bad idea to call commands with passwords in them and a few strategies you can use to design programs to avoid that (as well as a cool tip to keep commands out of the history file!)
+
+## Interactive examples
+
+### Notes
+
+```text
+Storing passwords/secrets:
+
+- configuration file
+- software for storing certificates, credentials, keys, passwords and any secrets in general:
+    - OS specific solution:
+        - Keyring / Seahorse (Linux)
+        - Keychain (MacOS)
+        - Credential Manager (Windows)
+        - Password Managers (generic software)
+    - Vault (self-hosted services)
+    - Cloud specific/hosted services
+- environment variable
+- standard input
+```
```

Comment:
Added additional note section for the different ways to store passwords/secrets that you described in the video.
I included some additional options for the software section.